src/main/scala/za/co/absa/commons/scalatest/spark/SparkFixture.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.scalatest.spark
+
+import org.apache.commons.io.FileUtils
+import org.apache.spark.sql.SparkSession
+import za.co.absa.commons.io.TempDirectory
+
+import java.io.File
+import java.sql.DriverManager
+import scala.util.Try
+
+trait SparkFixture {
+
+  val warehouseDir: String =
+    TempDirectory("SparkFixture", "UnitTest", pathOnly = true)
+      .deleteOnExit()
+      .path
+      .toString
+      .stripSuffix("/")
+
+  private val sessionBuilder: SparkSession.Builder =
+    SparkSession.builder
+      .master("local")
+      .config("spark.ui.enabled", "false")
+      .config("spark.sql.warehouse.dir", warehouseDir)
```

Comment:
This could be very helpful to set by default to run under VPN:
```scala
    .config("spark.driver.bindAddress","127.0.0.1")
    .config("spark.driver.host", "127.0.0.1")
```

Reply:
added